3-WebApp-graphapi-managed-identity/package.json

```diff
@@ -10,16 +10,15 @@
   "author": "derisen",
   "license": "MIT",
   "dependencies": {
-    "microsoft-identity-express": "git+https://github.com/Azure-Samples/microsoft-identity-express",
-    "@microsoft/microsoft-graph-client": "^2.2.1",
-    "@azure/identity": "^1.3.0",
-    "@azure/storage-blob": "^12.0.0",
+    "@azure/identity": "^3.0.0",
+    "@azure/storage-blob": "^12.11.0",
```

Comment:
Is @azure/storage-blob used in this project?

Reply:
good catch! removed